### PR TITLE
Stage UI Demo Content

### PR DESCRIPTION
**Pull request by MitchellShiell**
To Do:

- [x] Fix grid layout
- [x] Update and compress images following demo portal exploration page updates
- [x] Review edit and fix typos and typography in figures
- [x]  Update Footer CTA Links
- [x] Match CSS to documentation styling
- [x] Added notebox!
- [x] update formatting of content to be more readable
- [x] fix padding/margins between image title and text
- [x] Create public API key
- [x] add public key to this branch
- [ ] Test Public Key
- [x] Update download dropdown instruction link (may be easier to update documentation pages with demo variable information, the SPA about this portal does not have urls linking specific pages.. yet)

### CODE REVIEW

File: components/pages/about/ArticleComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { css, useTheme } from '@emotion/react';
+
+interface ArticleProps {
+	title?: string;
```
[the remark this answers is not in the document]
Comment (MitchellShiell) [reply components/pages/about/ArticleComponent.tsx:5]: It's sort of a hack but if I want a title, text, image, **text, image**, then having the title optional allows me to do that with two article components back to back

[the remark this answers is not in the document]
Comment (MitchellShiell) [reply components/pages/about/ArticleComponent.tsx:5]: never ended up needing that actually, set back to mandatory

File: components/pages/about/index.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const About = (): ReactElement => {
+	const [articleId, setArticleId] = useState<string>('overview');
+
+	const activeArticleId = (articleId: string) => setArticleId(articleId);
+
+	return (
+		<PageLayout subtitle="About This Portal">
+			<HeroBanner setArticleID={activeArticleId} activeId={articleId} />
+			<div>{activeArticleId}</div>
```
Comment (ciaranschutte) [inline components/pages/about/index.tsx:37]: this debugging line can be removed

File: components/pages/about/index.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+			<PageContent activeId={articleId} /> 
+		</PageLayout>
+	);
```
Comment (ciaranschutte) [inline components/pages/about/index.tsx:40]: The "white gap error" you see when there are 3 elements nested in the `PageLayout` is due to CSS.
The parent element on the page has `display: grid; grid-template-rows: 50px 1fr 47px`.

ref: https://github.com/overture-stack/dms-ui/blob/develop/components/PageLayout.tsx


So it knows how to layout 3 rows of content. 
The page header is the 1st row, the main page content is the 2nd row and the footer is the 3rd row.
You can check this out using web inspector tools (let me know if I can point you to the exact location!)

When you add extra elements our "3 rows grid" doesn't know how to layout 4...5.. etc elements. only 3.
To fix - make 1 element to contain all the content eg. a div... now we're back to 3 elements.

Comment (ciaranschutte) [reply components/pages/about/index.tsx:40]: two really good resources for more on grid (don't let the cutesyness put you off, both really really good resources): https://cssgridgarden.com and https://www.joshwcomeau.com/css/interactive-guide-to-grid/

**Review by ciaranschutte — CHANGES_REQUESTED** [review]

**Comment by ciaranschutte** [comment]
as discussed offline, performance will be improved once compressed images are implemented

File: components/pages/about/index.tsx (hunk 3 above)
Comment (MitchellShiell) [reply components/pages/about/index.tsx]: Complete!

File: tsconfig.json [hunk 4]
```diff
@@ -34,6 +34,12 @@
 			} // absolute paths for .d.ts files
 		]
 	},
-	"include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "additional.d.ts"],
+	"include": [
+		"next-env.d.ts",
+		"**/*.ts",
+		"**/*.tsx",
+		"additional.d.ts",
+		"components/pages/DocumentationContainerContainer"
```
Comment (ciaranschutte) [inline tsconfig.json:42]: Does this still exist?

Comment (MitchellShiell) [reply tsconfig.json:42]: Nope!

File: components/pages/disclaimers/HeroComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+ *  SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ *  TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ *  OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ *  IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+import React, { ReactElement } from 'react';
```
Comment (justincorrigible) [inline components/pages/disclaimers/HeroComponent.tsx:22]: We don't need to import React anymore
```suggestion
import { ReactElement } from 'react';
```

File: components/pages/disclaimers/notebox.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
+import React, { ReactNode } from 'react';
```
Comment (justincorrigible) [inline components/pages/disclaimers/notebox.tsx:1]:
```suggestion
import { ReactNode } from 'react';
```

File: pages/disclaimers/index.tsx [hunk 7]
```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
```
Comment (justincorrigible) [inline pages/disclaimers/index.tsx:3]: If we're creating new files, might well use the year that is done :)
```suggestion
 * Copyright (c) 2024 The Ontario Institute for Cancer Research. All rights reserved
```

File: components/pages/disclaimers/ArticleComponent.tsx [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+ *  SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ *  TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ *  OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ *  IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+import React from 'react';
```
Comment (justincorrigible) [inline components/pages/disclaimers/ArticleComponent.tsx:22]:
```suggestion
```

**Pull request closed without merging**